<!doctype html>
<!--[if lt IE 7]><html class="no-js lt-ie9 lt-ie8 lt-ie7" lang="en"> <![endif]-->
<!--[if (IE 7)&!(IEMobile)]><html class="no-js lt-ie9 lt-ie8" lang="en"><![endif]-->
<!--[if (IE 8)&!(IEMobile)]><html class="no-js lt-ie9" lang="en"><![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en"><!--<![endif]-->
<head>
<meta charset="utf-8">
<title>Tips for web developers &#8211; </title>

<meta name="keywords" content="">


<!-- Twitter Cards -->
<meta name="twitter:title" content="Tips for web developers">






<meta name="twitter:card" content="summary">
<meta name="twitter:image" content="http://localhost:4000/images/uchicago-mbl.png">
<meta property="og:image" content="http://localhost:4000/images/uchicago-mbl.png">


<!-- Open Graph -->
<meta property="og:locale" content="en_US">
<meta property="og:type" content="article">
<meta property="og:title" content="Tips for web developers">

<meta property="og:url" content="http://localhost:4000/web-tips/">
<meta property="og:site_name" content="Fast Genomics Solution">


<!-- <meta name="google-site-verification" content="x_a2aVtoLyANByc9VDBjyPm80IlGbNYUm4Vr1tSRhi0" /> -->


<link rel="canonical" href="http://localhost:4000/web-tips/">
<link href="http://localhost:4000/feed.xml" type="application/atom+xml" rel="alternate" title="Fast Genomics Solution Feed">

<meta name="HandheldFriendly" content="True">
<meta name="MobileOptimized" content="320">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<!-- <meta name="google-site-verification" content="x_a2aVtoLyANByc9VDBjyPm80IlGbNYUm4Vr1tSRhi0" /> -->

<!-- For all browsers -->
<link rel="stylesheet" href="/assets/css/main.css">

<meta http-equiv="cleartype" content="on">

<!-- HTML5 Shiv and Media Query Support -->
<!--[if lt IE 9]>
	<script src="http://localhost:4000/assets/js/vendor/html5shiv.min.js"></script>
	<script src="http://localhost:4000/assets/js/vendor/respond.min.js"></script>
<![endif]-->

<!-- Modernizr -->
<script src="http://localhost:4000/assets/js/vendor/modernizr-2.7.1.custom.min.js"></script>

<!-- MathJAX stuff -->
<script async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS_HTML-full">
  MathJax.Hub.Config({
    jax: ["input/TeX","output/HTML-CSS"],
    extensions: ["tex2jax.js"],
    tex2jax: {
      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
      processEscapes: true
    }
  });

  function doOnLoad() {
    var div = MathJax.HTML.Element(
          "div",
          {id: "MathDiv", style:{border:"1px solid", padding:"5px"}},
          ["Here is math: \\(x^2+1\\)",["br"],"and a display $$x+1\\over x-1$$"]
        );
    document.body.appendChild(div);
    // This does not seem needed but you can use if to force rendering of a div
    MathJax.Hub.Queue(["Typeset",MathJax.Hub,'MathDiv']);
  }
</script>

<!-- altmetric and dimensions -->
<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
<script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>

<link href='//fonts.googleapis.com/css?family=PT+Sans+Narrow:400,700%7CPT+Serif:400,700,400italic' rel='stylesheet' type='text/css'>

<!-- Zarul added -->
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Arvo">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Raleway">

<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Jersey+15&family=PT+Sans:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">


<!-- Icons -->
<!-- 16x16 -->
<link rel="shortcut icon" href="http://localhost:4000/favicon.ico">
<!-- 32x32 -->
<!-- <link rel="shortcut icon" href="http://localhost:4000/favicon.png"> -->
<!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
<link rel="apple-touch-icon-precomposed" href="http://localhost:4000/images/apple-touch-icon-precomposed.png">
<!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
<link rel="apple-touch-icon-precomposed" sizes="72x72" href="http://localhost:4000/images/apple-touch-icon-72x72-precomposed.png">
<!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
<link rel="apple-touch-icon-precomposed" sizes="114x114" href="http://localhost:4000/images/apple-touch-icon-114x114-precomposed.png">
<!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
<link rel="apple-touch-icon-precomposed" sizes="144x144" href="http://localhost:4000/images/apple-touch-icon-144x144-precomposed.png">

</head>

<body class="page">

<!--[if lt IE 9]><div class="browser-upgrade alert alert-info">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</div><![endif]-->


<a name="top"></a>
<div class="navigation-wrapper">
	<div class="site-name">
		<a href="http://localhost:4000/"><img src="/images/logo-company.png" alt="Home" style="width: 100%; height: auto;"></a>
	</div><!-- /.site-name -->
	<div class="top-navigation">
		<nav role="navigation" id="site-nav" class="nav">
		    <ul>
		        
				    
				    <li><a href="http://localhost:4000/aboutus/" style="color: #9e161a;" >About us</a></li>
				
				    
				    <li><a href="http://localhost:4000/services/" style="color: #9e161a;" >Services</a></li>
				
				    
				    <li><a href="http://localhost:4000/publications/" style="color: #9e161a;" >Publications</a></li>
				
				    
				    <li><a href="http://localhost:4000/posts/" style="color: #9e161a;" >Blog</a></li>
				
				    
				    <li><a href="http://localhost:4000/people/" style="color: #9e161a;" >People</a></li>
				
				    
				    <li><a href="http://localhost:4000/opportunities/" style="color: #9e161a;" >Opportunities</a></li>
				
				    
				    <li><a href="http://localhost:4000/contactus/" style="color: #9e161a;" >Contact us</a></li>
				
		    </ul>
		</nav>
	</div><!-- /.top-navigation -->
</div><!-- /.navigation-wrapper -->





<div id="main" role="main">

  <div class="article-author-side">
    
    <div>
  <p style="width: 150px;text-align: center;"><a href="https://fastgenomicssolutions.com/"><img src="/images/bg.jpg" style="border: none;" width="150"></a>
 <!--  <br /><a href="https://discord.gg/C6He6mSNY4
" _target="blank"><img class="discord-button-img-big" src="/images/discord.png" style="border: none;" /></a>

  <p style="width: 150px;text-align: left;"><a href="/momics/"><img src="/images/momics-teaching-material.png" style="border: none;" title="Microbial 'Omics Teaching Material"></a>
  <p style="width: 150px;text-align: left;"><a href="https://merenlab.org/2020/06/27/seminar-series-on-microbial-omics/"><img src="/images/momics-seminar-series.png" style="border: none;" title="Microbial 'Omics Seminar Series"></a>
  <p style="width: 150px;text-align: center;"><a href="http://localhost:4000/software/anvio-structure/"><img src="http://localhost:4000/images/anvio-structure-logo.png" style="border: none;" width="150"></a>
  <p><a href="/software/oligotyping/"><img src="http://localhost:4000/images/oligotyping-logo.png" style="border: none;"></a>
  <p><a href="/software/med/"><img src="http://localhost:4000/images/med-logo.png" style="border: none;" width="150"></a> -->
</div>

    
  </div>

  <article class="post">
    <h1>Tips for web developers</h1>
    
    <div class="article-wrap">

     <div id="articlecontent">

    
     <section id="markdown-toc" class="toc">
        <header><h3 id="toc-header"><i class="fa fa-book"></i> Table of Contents</h3></header>

        <div id="drawer"><script src="/assets/js/vendor/jquery-1.9.1.min.js"></script>
<script src='/assets/js/toc.js'></script>
<ul><li class="li-level-1"><a href="#showhide-content">Show/hide content</a></li><ul><li class="li-level-2"><a href="#information-boxes">Information boxes</a></li><li class="li-level-2"><a href="#warning-and-notice-statements">Warning and notice statements</a></li><li class="li-level-2"><a href="#turning-tab-delmited-files-to-markdown-notation">Turning TAB-delmited files to markdown notation</a></li><li class="li-level-2"><a href="#marking-anvio-artifacts-and-programs">Marking anvi’o artifacts and programs</a></li><li class="li-level-2"><a href="#fancy-quotes">Fancy quotes</a></li><li class="li-level-2"><a href="#images">Images</a></li><li class="li-level-2"><a href="#a-useful-shortcut-for-vim-users">A useful shortcut for vim users</a></li></ul></ul>
</div>
    </section>
    

    <p>The purpose of this page is to show some of the fancy things our web page can do. If you add new features that are specific to our web code, or discover uses of features that are not described here, please feel free to extend this list.</p>

<h1 id="showhide-content">Show/hide content</h1>

<p>If you want to show/hide content, you can use this notation in your markdown files.</p>

<p><strong>Raw</strong>:</p>

<div class="language-html highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
<span class="nt">&lt;details</span> <span class="na">markdown=</span><span class="s">"1"</span><span class="nt">&gt;&lt;summary&gt;</span>Show/Hide SOME CONTENT<span class="nt">&lt;/summary&gt;</span>

SOME CONTENT GOES HERE

<span class="nt">&lt;/details&gt;</span>

</code></pre></div></div>

<p><strong>Rendered</strong>:</p>

<details>
  <summary>Show/Hide SOME CONTENT</summary>

  <p>SOME CONTENT GOES HERE</p>

</details>

<h2 id="information-boxes">Information boxes</h2>

<p>If you want to show summary sections with a different background color, you can use this notation in your markdown files.</p>

<p><strong>Raw</strong>:</p>

<div class="language-html highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
<span class="nt">&lt;div</span> <span class="na">class=</span><span class="s">"extra-info"</span> <span class="na">markdown=</span><span class="s">"1"</span><span class="nt">&gt;</span>

<span class="nt">&lt;span</span> <span class="na">class=</span><span class="s">"extra-info-header"</span><span class="nt">&gt;</span>Smart title for extra info<span class="nt">&lt;/span&gt;</span>

EXTRA INFO GOES HERE

<span class="nt">&lt;/div&gt;</span>

</code></pre></div></div>

<p><strong>Rendered</strong>:</p>

<div class="extra-info">

  <p><span class="extra-info-header">Smart title for extra info</span></p>

  <p>EXTRA INFO GOES HERE</p>

</div>

<h2 id="warning-and-notice-statements">Warning and notice statements</h2>

<p>You should feel free to use warning and notice statements.</p>

<p><strong>Raw</strong>:</p>

<div class="language-html highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
{:.warning}
A warning messages goes here.

{:.notice}
A notice statement goes here.

</code></pre></div></div>

<p><strong>Rendered</strong>:</p>

<p class="warning">A warning messages goes here.</p>

<p class="notice">A notice statement goes here.</p>

<h2 id="turning-tab-delmited-files-to-markdown-notation">Turning TAB-delmited files to markdown notation</h2>

<p>Tutorial writers often need to display contents of TAB-delimited files. One easy way to do that is to use the program <span class="artifact-p"><a href="https://anvio.org/help/main/programs/anvi-script-as-markdown" target="_blank">anvi-script-as-markdown</a></span>. See examples on the program page.</p>

<h2 id="marking-anvio-artifacts-and-programs">Marking anvi’o artifacts and programs</h2>

<p>You can use this notation to link anvi’o programs and artifacts to their documentation with fancy looking links. If you are a <code class="language-plaintext highlighter-rouge">vim</code> user, you can automatize this conversion. Please see the section in this file called “A useful shortcut for vim users”.</p>

<p><strong>Raw</strong>:</p>

<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
In all fairness, an anvi'o {% include ARTIFACT name="contigs-db" text="contigs database" %} is just a FASTA file adjusted to 2020 and is generated by {% include PROGRAM name="anvi-gen-contigs-database" text="a fancy anvi'o program" %}.

</code></pre></div></div>

<p><strong>Rendered</strong>:</p>

<p>In all fairness, an anvi’o <span class="artifact-r"><a href="https://anvio.org/help/main/artifacts/contigs-db">contigs database</a></span> is just a FASTA file adjusted to 2020 and is generated by <span class="artifact-p"><a href="https://anvio.org/help/main/programs/anvi-gen-contigs-database" target="_blank">a fancy anvi’o program</a></span>.</p>

<p><strong>Raw</strong>:</p>

<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
An anvi'o {% include ARTIFACT name="pan-db" %} comes from {% include PROGRAM name="anvi-pan-genome" %}.

</code></pre></div></div>

<p><strong>Rendered</strong>:</p>

<p>An anvi’o <span class="artifact-r"><a href="https://anvio.org/help/main/artifacts/pan-db">pan-db</a></span> comes from <span class="artifact-p"><a href="https://anvio.org/help/main/programs/anvi-pan-genome" target="_blank">anvi-pan-genome</a></span>.</p>

<h2 id="fancy-quotes">Fancy quotes</h2>

<p><strong>Raw</strong>:</p>

<div class="language-html highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="nt">&lt;blockquote&gt;</span>
An intersting quote.
<span class="nt">&lt;div</span> <span class="na">class=</span><span class="s">"blockquote-author"</span><span class="nt">&gt;</span>Someone Interesting<span class="nt">&lt;/div&gt;</span>
<span class="nt">&lt;/blockquote&gt;</span>
</code></pre></div></div>

<p><strong>Rendered</strong>:</p>

<blockquote>
An intersting quote.
<div class="blockquote-author">Someone Interesting</div>
</blockquote>

<h2 id="images">Images</h2>

<p>Never load your images directly, and always use this notation, which will enable lazy loading and will standardize all images:</p>

<div class="language-html highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
{% include IMAGE path="/path/to/image.png" width=50 %}

</code></pre></div></div>

<p>Here are some examples:</p>

<div class="language-html highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
{% include IMAGE path="https://merenlab.org/logo.png" %}

</code></pre></div></div>

<div class="language-html highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
{% include IMAGE path="some_folder/store_c.gif" width=50 %}

</code></pre></div></div>

<p>You can add title text for your figures, which will appear as a tiny text when people hover their mouse pointers:</p>

<div class="language-html highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
{% include IMAGE path="/images/ref_bg.gif" title="Changing the background color of a layer" width=90 %}

</code></pre></div></div>

<p>You can also add captions to your figures, which will appear as italicized text closer to your figure:</p>

<div class="language-html highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
{% include IMAGE path="/images/some_image.gif" caption="A descriptive short paragraph to explain what this figure is showing" %}

</code></pre></div></div>

<h2 id="a-useful-shortcut-for-vim-users">A useful shortcut for vim users</h2>

<p>If you are using vim as a text editor, you can automatically turn anvi’o program and artifact names into nice markings shown below instantenously with no effort as originally suggested by Evan Kiefl. With this solution, when you type the following in any document in <code class="language-plaintext highlighter-rouge">insert</code> mode (notice the extra <code class="language-plaintext highlighter-rouge">i</code> at the end of program and artifact names),</p>

<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>An anvi'o pan-dbi comes from anvi-pan-genomei.
</code></pre></div></div>

<p>vim will automatically turn it into the following as you write:</p>

<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
An anvi'o {% include ARTIFACT name="pan-db" %} comes from {% include PROGRAM name="anvi-pan-genome" %}.

</code></pre></div></div>

<p>Which will then be rendered as “An anvi’o <span class="artifact-r"><a href="https://anvio.org/help/main/artifacts/pan-db">pan-db</a></span> comes from <span class="artifact-p"><a href="https://anvio.org/help/main/programs/anvi-pan-genome" target="_blank">anvi-pan-genome</a></span>”, as you know.</p>

<p>For this to improve your tutorial writing, all you need to do is to add the following lines somewhere in your <code class="language-plaintext highlighter-rouge">~/.vimrc</code> file:</p>

<div class="language-vim highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
<span class="c">" Anvi'o artifacts</span>
inoreabbrev pan<span class="p">-</span>dbi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"pan-db"</span> %<span class="p">}</span>
inoreabbrev contigs<span class="p">-</span>dbi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"contigs-db"</span> %<span class="p">}</span>
inoreabbrev trnaseq<span class="p">-</span>dbi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"trnaseq-db"</span> %<span class="p">}</span>
inoreabbrev trnaseq<span class="p">-</span>contigs<span class="p">-</span>dbi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"trnaseq-contigs-db"</span> %<span class="p">}</span>
inoreabbrev trnaseq<span class="p">-</span><span class="k">profile</span><span class="p">-</span>dbi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"trnaseq-profile-db"</span> %<span class="p">}</span>
inoreabbrev modules<span class="p">-</span>dbi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"modules-db"</span> %<span class="p">}</span>
inoreabbrev fastai <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"fasta"</span> %<span class="p">}</span>
inoreabbrev contigs<span class="p">-</span>fastai <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"contigs-fasta"</span> %<span class="p">}</span>
inoreabbrev dna<span class="p">-</span>sequencei <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"dna-sequence"</span> %<span class="p">}</span>
inoreabbrev trnaseq<span class="p">-</span>fastai <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"trnaseq-fasta"</span> %<span class="p">}</span>
inoreabbrev configuration<span class="p">-</span>inii <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"configuration-ini"</span> %<span class="p">}</span>
inoreabbrev external<span class="p">-</span>gene<span class="p">-</span>callsi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"external-gene-calls"</span> %<span class="p">}</span>
inoreabbrev external<span class="p">-</span>structuresi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"external-structures"</span> %<span class="p">}</span>
inoreabbrev concatenated<span class="p">-</span>gene<span class="p">-</span>alignment<span class="p">-</span>fastai <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"concatenated-gene-alignment-fasta"</span> %<span class="p">}</span>
inoreabbrev short<span class="p">-</span>reads<span class="p">-</span>fastai <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"short-reads-fasta"</span> %<span class="p">}</span>
inoreabbrev genes<span class="p">-</span>fastai <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"genes-fasta"</span> %<span class="p">}</span>
inoreabbrev bam<span class="p">-</span>filei <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"bam-file"</span> %<span class="p">}</span>
inoreabbrev bam<span class="p">-</span>stats<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"bam-stats-txt"</span> %<span class="p">}</span>
inoreabbrev protein<span class="p">-</span>structure<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"protein-structure-txt"</span> %<span class="p">}</span>
inoreabbrev samples<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"samples-txt"</span> %<span class="p">}</span>
inoreabbrev primers<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"primers-txt"</span> %<span class="p">}</span>
inoreabbrev fasta<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"fasta-txt"</span> %<span class="p">}</span>
inoreabbrev raw<span class="p">-</span>bam<span class="p">-</span>filei <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"raw-bam-file"</span> %<span class="p">}</span>
inoreabbrev locus<span class="p">-</span>fastai <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"locus-fasta"</span> %<span class="p">}</span>
inoreabbrev structure<span class="p">-</span>dbi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"structure-db"</span> %<span class="p">}</span>
inoreabbrev pdb<span class="p">-</span>dbi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"pdb-db"</span> %<span class="p">}</span>
inoreabbrev kegg<span class="p">-</span>datai <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"kegg-data"</span> %<span class="p">}</span>
inoreabbrev single<span class="p">-</span><span class="k">profile</span><span class="p">-</span>dbi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"single-profile-db"</span> %<span class="p">}</span>
inoreabbrev <span class="k">profile</span><span class="p">-</span>dbi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"profile-db"</span> %<span class="p">}</span>
inoreabbrev genes<span class="p">-</span>dbi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"genes-db"</span> %<span class="p">}</span>
inoreabbrev genomes<span class="p">-</span>storage<span class="p">-</span>dbi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"genomes-storage-db"</span> %<span class="p">}</span>
inoreabbrev contigs<span class="p">-</span>statsi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"contigs-stats"</span> %<span class="p">}</span>
inoreabbrev svgi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"svg"</span> %<span class="p">}</span>
inoreabbrev bini <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"bin"</span> %<span class="p">}</span>
inoreabbrev collectioni <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"collection"</span> %<span class="p">}</span>
inoreabbrev collection<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"collection-txt"</span> %<span class="p">}</span>
inoreabbrev hmm<span class="p">-</span>sourcei <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"hmm-source"</span> %<span class="p">}</span>
inoreabbrev hmm<span class="p">-</span>hitsi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"hmm-hits"</span> %<span class="p">}</span>
inoreabbrev completioni <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"completion"</span> %<span class="p">}</span>
inoreabbrev cogs<span class="p">-</span>datai <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"cogs-data"</span> %<span class="p">}</span>
inoreabbrev pfams<span class="p">-</span>datai <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"pfams-data"</span> %<span class="p">}</span>
inoreabbrev misc<span class="p">-</span>data<span class="p">-</span><span class="nb">items</span><span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"misc-data-items-txt"</span> %<span class="p">}</span>
inoreabbrev misc<span class="p">-</span>data<span class="p">-</span>itemsi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"misc-data-items"</span> %<span class="p">}</span>
inoreabbrev misc<span class="p">-</span>data<span class="p">-</span>layers<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"misc-data-layers-txt"</span> %<span class="p">}</span>
inoreabbrev misc<span class="p">-</span>data<span class="p">-</span>layersi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"misc-data-layers"</span> %<span class="p">}</span>
inoreabbrev misc<span class="p">-</span>data<span class="p">-</span>nucleotides<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"misc-data-nucleotides-txt"</span> %<span class="p">}</span>
inoreabbrev misc<span class="p">-</span>data<span class="p">-</span>nucleotidesi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"misc-data-nucleotides"</span> %<span class="p">}</span>
inoreabbrev misc<span class="p">-</span>data<span class="p">-</span>amino<span class="p">-</span>acids<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"misc-data-amino-acids-txt"</span> %<span class="p">}</span>
inoreabbrev misc<span class="p">-</span>data<span class="p">-</span>amino<span class="p">-</span>acidsi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"misc-data-amino-acids"</span> %<span class="p">}</span>
inoreabbrev genome<span class="p">-</span>similarityi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"genome-similarity"</span> %<span class="p">}</span>
inoreabbrev misc<span class="p">-</span>data<span class="p">-</span>layer<span class="p">-</span>orders<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"misc-data-layer-orders-txt"</span> %<span class="p">}</span>
inoreabbrev misc<span class="p">-</span>data<span class="p">-</span>layer<span class="p">-</span>ordersi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"misc-data-layer-orders"</span> %<span class="p">}</span>
inoreabbrev misc<span class="p">-</span>data<span class="p">-</span><span class="nb">items</span><span class="p">-</span>order<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"misc-data-items-order-txt"</span> %<span class="p">}</span>
inoreabbrev misc<span class="p">-</span>data<span class="p">-</span><span class="nb">items</span><span class="p">-</span>orderi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"misc-data-items-order"</span> %<span class="p">}</span>
inoreabbrev dendrogrami <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"dendrogram"</span> %<span class="p">}</span>
inoreabbrev metapangenomei <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"metapangenome"</span> %<span class="p">}</span>
inoreabbrev oligotypesi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"oligotypes"</span> %<span class="p">}</span>
inoreabbrev linkmers<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"linkmers-txt"</span> %<span class="p">}</span>
inoreabbrev palindromes<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"palindromes-txt"</span> %<span class="p">}</span>
inoreabbrev inversionsi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"inversions"</span> %<span class="p">}</span>
inoreabbrev phylogenyi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"phylogeny"</span> %<span class="p">}</span>
inoreabbrev gene<span class="p">-</span>calls<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"gene-calls-txt"</span> %<span class="p">}</span>
inoreabbrev binding<span class="p">-</span>frequencies<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"binding-frequencies-txt"</span> %<span class="p">}</span>
inoreabbrev interacdome<span class="p">-</span>datai <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"interacdome-data"</span> %<span class="p">}</span>
inoreabbrev functionsi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"functions"</span> %<span class="p">}</span>
inoreabbrev functions<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"functions-txt"</span> %<span class="p">}</span>
inoreabbrev functional<span class="p">-</span>enrichment<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"functional-enrichment-txt"</span> %<span class="p">}</span>
inoreabbrev kegg<span class="p">-</span>functionsi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"kegg-functions"</span> %<span class="p">}</span>
inoreabbrev interactivei <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"interactive"</span> %<span class="p">}</span>
inoreabbrev <span class="k">view</span><span class="p">-</span>datai <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"view-data"</span> %<span class="p">}</span>
inoreabbrev layer<span class="p">-</span>taxonomyi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"layer-taxonomy"</span> %<span class="p">}</span>
inoreabbrev layer<span class="p">-</span>taxonomy<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"layer-taxonomy-txt"</span> %<span class="p">}</span>
inoreabbrev gene<span class="p">-</span>taxonomyi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"gene-taxonomy"</span> %<span class="p">}</span>
inoreabbrev gene<span class="p">-</span>taxonomy<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"gene-taxonomy-txt"</span> %<span class="p">}</span>
inoreabbrev genome<span class="p">-</span>taxonomyi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"genome-taxonomy"</span> %<span class="p">}</span>
inoreabbrev genome<span class="p">-</span>taxonomy<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"genome-taxonomy-txt"</span> %<span class="p">}</span>
inoreabbrev scgs<span class="p">-</span>taxonomy<span class="p">-</span>dbi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"scgs-taxonomy-db"</span> %<span class="p">}</span>
inoreabbrev scgs<span class="p">-</span>taxonomyi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"scgs-taxonomy"</span> %<span class="p">}</span>
inoreabbrev trna<span class="p">-</span>taxonomy<span class="p">-</span>dbi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"trna-taxonomy-db"</span> %<span class="p">}</span>
inoreabbrev trna<span class="p">-</span>taxonomyi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"trna-taxonomy"</span> %<span class="p">}</span>
inoreabbrev external<span class="p">-</span>genomesi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"external-genomes"</span> %<span class="p">}</span>
inoreabbrev internal<span class="p">-</span>genomesi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"internal-genomes"</span> %<span class="p">}</span>
inoreabbrev metagenomesi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"metagenomes"</span> %<span class="p">}</span>
inoreabbrev coverages<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"coverages-txt"</span> %<span class="p">}</span>
inoreabbrev detection<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"detection-txt"</span> %<span class="p">}</span>
inoreabbrev variability<span class="p">-</span>profilei <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"variability-profile"</span> %<span class="p">}</span>
inoreabbrev variability<span class="p">-</span><span class="k">profile</span><span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"variability-profile-txt"</span> %<span class="p">}</span>
inoreabbrev codon<span class="p">-</span>frequencies<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"codon-frequencies-txt"</span> %<span class="p">}</span>
inoreabbrev aa<span class="p">-</span>frequencies<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"aa-frequencies-txt"</span> %<span class="p">}</span>
inoreabbrev fixation<span class="p">-</span><span class="nb">index</span><span class="p">-</span>matrixi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"fixation-index-matrix"</span> %<span class="p">}</span>
inoreabbrev trnaseq<span class="p">-</span>seed<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"trnaseq-seed-txt"</span> %<span class="p">}</span>
inoreabbrev seeds<span class="p">-</span>specific<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"seeds-specific-txt"</span> %<span class="p">}</span>
inoreabbrev seeds<span class="p">-</span>non<span class="p">-</span>specific<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"seeds-non-specific-txt"</span> %<span class="p">}</span>
inoreabbrev modifications<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"modifications-txt"</span> %<span class="p">}</span>
inoreabbrev trnaseq<span class="p">-</span>ploti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"trnaseq-plot"</span> %<span class="p">}</span>
inoreabbrev summaryi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"summary"</span> %<span class="p">}</span>
inoreabbrev quick<span class="p">-</span>summaryi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"quick-summary"</span> %<span class="p">}</span>
inoreabbrev <span class="k">split</span><span class="p">-</span>binsi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"split-bins"</span> %<span class="p">}</span>
inoreabbrev statei <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"state"</span> %<span class="p">}</span>
inoreabbrev ngramsi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"ngrams"</span> %<span class="p">}</span>
inoreabbrev <span class="nb">state</span><span class="p">-</span>jsoni <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"state-json"</span> %<span class="p">}</span>
inoreabbrev kegg<span class="p">-</span>metabolismi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"kegg-metabolism"</span> %<span class="p">}</span>
inoreabbrev augustus<span class="p">-</span>gene<span class="p">-</span>callsi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"augustus-gene-calls"</span> %<span class="p">}</span>
inoreabbrev genes<span class="p">-</span>statsi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"genes-stats"</span> %<span class="p">}</span>
inoreabbrev vcfi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"vcf"</span> %<span class="p">}</span>
inoreabbrev <span class="k">blast</span><span class="p">-</span>tablei <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"blast-table"</span> %<span class="p">}</span>
inoreabbrev splits<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"splits-txt"</span> %<span class="p">}</span>
inoreabbrev genbank<span class="p">-</span>filei <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"genbank-file"</span> %<span class="p">}</span>
inoreabbrev groups<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"groups-txt"</span> %<span class="p">}</span>
inoreabbrev splits<span class="p">-</span>taxonomy<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"splits-taxonomy-txt"</span> %<span class="p">}</span>
inoreabbrev hmm<span class="p">-</span>hits<span class="p">-</span>matrix<span class="p">-</span>txti <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"hmm-hits-matrix-txt"</span> %<span class="p">}</span>
inoreabbrev pn<span class="p">-</span><span class="k">ps</span><span class="p">-</span>datai <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"pn-ps-data"</span> %<span class="p">}</span>
inoreabbrev clustering<span class="p">-</span>configurationi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"clustering-configuration"</span> %<span class="p">}</span>
inoreabbrev workflow<span class="p">-</span>configi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"workflow-config"</span> %<span class="p">}</span>
inoreabbrev contigs<span class="p">-</span>workflowi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"contigs-workflow"</span> %<span class="p">}</span>
inoreabbrev metagenomics<span class="p">-</span>workflowi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"metagenomics-workflow"</span> %<span class="p">}</span>
inoreabbrev pangenomics<span class="p">-</span>workflowi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"pangenomics-workflow"</span> %<span class="p">}</span>
inoreabbrev phylogenomics<span class="p">-</span>workflowi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"phylogenomics-workflow"</span> %<span class="p">}</span>
inoreabbrev trnaseq<span class="p">-</span>workflowi <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"trnaseq-workflow"</span> %<span class="p">}</span>
inoreabbrev contig<span class="p">-</span>inspectioni <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"contig-inspection"</span> %<span class="p">}</span>
inoreabbrev gene<span class="p">-</span>cluster<span class="p">-</span>inspectioni <span class="p">{</span>% <span class="nb">include</span> ARTIFACT name<span class="p">=</span><span class="s2">"gene-cluster-inspection"</span> %<span class="p">}</span>

<span class="c">" Anvi'o programs</span>
inoreabbrev anvi<span class="p">-</span>analyze<span class="p">-</span>syntenyi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-analyze-synteny"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>cluster<span class="p">-</span>contigsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-cluster-contigs"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>compute<span class="p">-</span>anii <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-compute-ani"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>compute<span class="p">-</span>completenessi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-compute-completeness"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>compute<span class="p">-</span>functional<span class="p">-</span>enrichmenti <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-compute-functional-enrichment"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>compute<span class="p">-</span>functional<span class="p">-</span>enrichment<span class="p">-</span>across<span class="p">-</span>genomesi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-compute-functional-enrichment-across-genomes"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>compute<span class="p">-</span>functional<span class="p">-</span>enrichment<span class="p">-</span><span class="k">in</span><span class="p">-</span>pani <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-compute-functional-enrichment-in-pan"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>compute<span class="p">-</span>gene<span class="p">-</span>cluster<span class="p">-</span>homogeneityi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-compute-gene-cluster-homogeneity"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>compute<span class="p">-</span>genome<span class="p">-</span>similarityi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-compute-genome-similarity"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>compute<span class="p">-</span>metabolic<span class="p">-</span>enrichmenti <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-compute-metabolic-enrichment"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>db<span class="p">-</span>infoi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-db-info"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">delete</span><span class="p">-</span>collectioni <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-delete-collection"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">delete</span><span class="p">-</span>functionsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-delete-functions"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">delete</span><span class="p">-</span>hmmsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-delete-hmms"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">delete</span><span class="p">-</span>misc<span class="p">-</span>datai <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-delete-misc-data"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">delete</span><span class="p">-</span>statei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-delete-state"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>dereplicate<span class="p">-</span>genomesi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-dereplicate-genomes"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">display</span><span class="p">-</span>contigs<span class="p">-</span>statsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-display-contigs-stats"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">display</span><span class="p">-</span>functionsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-display-functions"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">display</span><span class="p">-</span>metabolismi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-display-metabolism"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">display</span><span class="p">-</span>pani <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-display-pan"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">display</span><span class="p">-</span>structurei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-display-structure"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>estimate<span class="p">-</span>genome<span class="p">-</span>completenessi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-estimate-genome-completeness"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>estimate<span class="p">-</span>genome<span class="p">-</span>taxonomyi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-estimate-genome-taxonomy"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>estimate<span class="p">-</span>metabolismi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-estimate-metabolism"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>estimate<span class="p">-</span><span class="k">scg</span><span class="p">-</span>taxonomyi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-estimate-scg-taxonomy"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>estimate<span class="p">-</span>trna<span class="p">-</span>taxonomyi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-estimate-trna-taxonomy"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>experimental<span class="p">-</span>organizationi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-experimental-organization"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">export</span><span class="p">-</span>collectioni <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-export-collection"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">export</span><span class="p">-</span>contigsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-export-contigs"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">export</span><span class="p">-</span>functionsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-export-functions"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">export</span><span class="p">-</span>gene<span class="p">-</span>callsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-export-gene-calls"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">export</span><span class="p">-</span>gene<span class="p">-</span>coverage<span class="p">-</span><span class="nb">and</span><span class="p">-</span>detectioni <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-export-gene-coverage-and-detection"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">export</span><span class="p">-</span><span class="nb">items</span><span class="p">-</span>orderi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-export-items-order"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">export</span><span class="p">-</span>locusi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-export-locus"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">export</span><span class="p">-</span>misc<span class="p">-</span>datai <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-export-misc-data"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">export</span><span class="p">-</span>splits<span class="p">-</span><span class="nb">and</span><span class="p">-</span>coveragesi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-export-splits-and-coverages"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">export</span><span class="p">-</span>splits<span class="p">-</span>taxonomyi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-export-splits-taxonomy"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">export</span><span class="p">-</span>statei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-export-state"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">export</span><span class="p">-</span>structuresi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-export-structures"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">export</span><span class="p">-</span>tablei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-export-table"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>gen<span class="p">-</span>contigs<span class="p">-</span>databasei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-gen-contigs-database"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>gen<span class="p">-</span>fixation<span class="p">-</span><span class="nb">index</span><span class="p">-</span>matrixi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-gen-fixation-index-matrix"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>gen<span class="p">-</span>gene<span class="p">-</span>consensus<span class="p">-</span>sequencesi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-gen-gene-consensus-sequences"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>gen<span class="p">-</span>gene<span class="p">-</span>level<span class="p">-</span>stats<span class="p">-</span>databasesi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-gen-gene-level-stats-databases"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>gen<span class="p">-</span>genomes<span class="p">-</span>storagei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-gen-genomes-storage"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>gen<span class="p">-</span>networki <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-gen-network"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>gen<span class="p">-</span>phylogenomic<span class="p">-</span>treei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-gen-phylogenomic-tree"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>gen<span class="p">-</span>structure<span class="p">-</span>databasei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-gen-structure-database"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>gen<span class="p">-</span>variability<span class="p">-</span>matrixi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-gen-variability-matrix"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>gen<span class="p">-</span>variability<span class="p">-</span>networki <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-gen-variability-network"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>gen<span class="p">-</span>variability<span class="p">-</span>profilei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-gen-variability-profile"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="nb">get</span><span class="p">-</span>aa<span class="p">-</span>countsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-get-aa-counts"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="nb">get</span><span class="p">-</span>codon<span class="p">-</span>frequenciesi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-get-codon-frequencies"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="nb">get</span><span class="p">-</span>pn<span class="p">-</span><span class="k">ps</span><span class="p">-</span>ratioi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-get-pn-ps-ratio"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="nb">get</span><span class="p">-</span>sequences<span class="p">-</span><span class="k">for</span><span class="p">-</span>gene<span class="p">-</span>callsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-get-sequences-for-gene-calls"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="nb">get</span><span class="p">-</span>sequences<span class="p">-</span><span class="k">for</span><span class="p">-</span>gene<span class="p">-</span>clustersi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-get-sequences-for-gene-clusters"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="nb">get</span><span class="p">-</span>sequences<span class="p">-</span><span class="k">for</span><span class="p">-</span>hmm<span class="p">-</span>hitsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-get-sequences-for-hmm-hits"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="nb">get</span><span class="p">-</span>short<span class="p">-</span>reads<span class="p">-</span>from<span class="p">-</span>bami <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-get-short-reads-from-bam"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="nb">get</span><span class="p">-</span>short<span class="p">-</span>reads<span class="p">-</span>mapping<span class="p">-</span><span class="k">to</span><span class="p">-</span><span class="k">a</span><span class="p">-</span>genei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-get-short-reads-mapping-to-a-gene"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="nb">get</span><span class="p">-</span><span class="k">split</span><span class="p">-</span>coveragesi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-get-split-coverages"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="nb">get</span><span class="p">-</span>tlen<span class="p">-</span>dist<span class="p">-</span>from<span class="p">-</span>bami <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-get-tlen-dist-from-bam"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>helpi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-help"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">import</span><span class="p">-</span>collectioni <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-import-collection"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">import</span><span class="p">-</span>functionsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-import-functions"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">import</span><span class="p">-</span><span class="nb">items</span><span class="p">-</span>orderi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-import-items-order"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">import</span><span class="p">-</span>misc<span class="p">-</span>datai <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-import-misc-data"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">import</span><span class="p">-</span>statei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-import-state"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">import</span><span class="p">-</span>taxonomy<span class="p">-</span><span class="k">for</span><span class="p">-</span>genesi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-import-taxonomy-for-genes"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">import</span><span class="p">-</span>taxonomy<span class="p">-</span><span class="k">for</span><span class="p">-</span>layersi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-import-taxonomy-for-layers"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>init<span class="p">-</span>bami <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-init-bam"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>inspecti <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-inspect"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>interactivei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-interactive"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>matrix<span class="p">-</span><span class="k">to</span><span class="p">-</span>newicki <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-matrix-to-newick"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>mcg<span class="p">-</span>classifieri <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-mcg-classifier"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>mergei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-merge"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>merge<span class="p">-</span>binsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-merge-bins"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>merge<span class="p">-</span>trnaseqi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-merge-trnaseq"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>meta<span class="p">-</span>pan<span class="p">-</span>genomei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-meta-pan-genome"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>migratei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-migrate"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>oligotype<span class="p">-</span>linkmersi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-oligotype-linkmers"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>pan<span class="p">-</span>genomei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-pan-genome"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>plot<span class="p">-</span>trnaseqi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-plot-trnaseq"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>profilei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-profile"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">profile</span><span class="p">-</span>blitzi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-profile-blitz"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>pushi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-push"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>refinei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-refine"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="nb">rename</span><span class="p">-</span>binsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-rename-bins"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="nb">report</span><span class="p">-</span>inversionsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-report-inversions"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="nb">report</span><span class="p">-</span>linkmersi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-report-linkmers"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>run<span class="p">-</span>hmmsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-run-hmms"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>run<span class="p">-</span>interacdomei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-run-interacdome"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>run<span class="p">-</span>kegg<span class="p">-</span>kofamsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-run-kegg-kofams"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>run<span class="p">-</span>ncbi<span class="p">-</span>cogsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-run-ncbi-cogs"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>run<span class="p">-</span>pfamsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-run-pfams"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>run<span class="p">-</span><span class="k">scg</span><span class="p">-</span>taxonomyi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-run-scg-taxonomy"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>run<span class="p">-</span>trna<span class="p">-</span>taxonomyi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-run-trna-taxonomy"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>run<span class="p">-</span>workflowi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-run-workflow"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>scan<span class="p">-</span>trnasi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-scan-trnas"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="nb">search</span><span class="p">-</span>functionsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-search-functions"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="nb">search</span><span class="p">-</span>palindromesi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-search-palindromes"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="nb">search</span><span class="p">-</span>sequence<span class="p">-</span>motifsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-search-sequence-motifs"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>self<span class="p">-</span>testi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-self-test"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>setup<span class="p">-</span>interacdomei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-setup-interacdome"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>setup<span class="p">-</span>kegg<span class="p">-</span>kofamsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-setup-kegg-kofams"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>setup<span class="p">-</span>ncbi<span class="p">-</span>cogsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-setup-ncbi-cogs"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>setup<span class="p">-</span>pdb<span class="p">-</span>databasei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-setup-pdb-database"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>setup<span class="p">-</span>pfamsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-setup-pfams"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>setup<span class="p">-</span><span class="k">scg</span><span class="p">-</span>taxonomyi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-setup-scg-taxonomy"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>setup<span class="p">-</span>trna<span class="p">-</span>taxonomyi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-setup-trna-taxonomy"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>show<span class="p">-</span>collections<span class="p">-</span><span class="nb">and</span><span class="p">-</span>binsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-show-collections-and-bins"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>show<span class="p">-</span>misc<span class="p">-</span>datai <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-show-misc-data"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>spliti <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-split"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>summarizei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-summarize"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>summarize<span class="p">-</span>blitzi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-summarize-blitz"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>tabulate<span class="p">-</span>trnaseqi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-tabulate-trnaseq"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>trnaseqi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-trnaseq"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">update</span><span class="p">-</span>db<span class="p">-</span>descriptioni <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-update-db-description"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span><span class="k">update</span><span class="p">-</span>structure<span class="p">-</span>databasei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-update-structure-database"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>upgradei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-upgrade"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span><span class="nb">add</span><span class="p">-</span>default<span class="p">-</span>collectioni <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-add-default-collection"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span><span class="k">as</span><span class="p">-</span>markdowni <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-as-markdown"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>checkm<span class="p">-</span>tree<span class="p">-</span><span class="k">to</span><span class="p">-</span>interactivei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-checkm-tree-to-interactive"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>compute<span class="p">-</span>ani<span class="p">-</span><span class="k">for</span><span class="p">-</span>fastai <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-compute-ani-for-fasta"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>compute<span class="p">-</span>bayesian<span class="p">-</span>pan<span class="p">-</span>corei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-compute-bayesian-pan-core"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>enrichment<span class="p">-</span>statsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-enrichment-stats"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>estimate<span class="p">-</span>genome<span class="p">-</span>sizei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-estimate-genome-size"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span><span class="k">filter</span><span class="p">-</span>fasta<span class="p">-</span>by<span class="p">-</span>blasti <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-filter-fasta-by-blast"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span><span class="k">filter</span><span class="p">-</span>hmm<span class="p">-</span>hits<span class="p">-</span>tablei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-filter-hmm-hits-table"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span><span class="k">fix</span><span class="p">-</span>homopolymer<span class="p">-</span>indelsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-fix-homopolymer-indels"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>gen<span class="p">-</span>CPR<span class="p">-</span>classifieri <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-gen-CPR-classifier"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>gen<span class="p">-</span>distribution<span class="p">-</span>of<span class="p">-</span>genes<span class="p">-</span><span class="k">in</span><span class="p">-</span><span class="k">a</span><span class="p">-</span>bini <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-gen-distribution-of-genes-in-a-bin"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>gen<span class="p">-</span>functions<span class="p">-</span>per<span class="p">-</span>group<span class="p">-</span>stats<span class="p">-</span>outputi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-gen-functions-per-group-stats-output"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>gen<span class="p">-</span>genomes<span class="p">-</span>filei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-gen-genomes-file"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>gen<span class="p">-</span><span class="k">help</span><span class="p">-</span>pagesi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-gen-help-pages"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>gen<span class="p">-</span>hmm<span class="p">-</span>hits<span class="p">-</span>matrix<span class="p">-</span>across<span class="p">-</span>genomesi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-gen-hmm-hits-matrix-across-genomes"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>gen<span class="p">-</span>programs<span class="p">-</span>networki <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-gen-programs-network"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>gen<span class="p">-</span>programs<span class="p">-</span>vignettei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-gen-programs-vignette"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>gen<span class="p">-</span>pseudo<span class="p">-</span>paired<span class="p">-</span>reads<span class="p">-</span>from<span class="p">-</span>fastqi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-gen-pseudo-paired-reads-from-fastq"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>gen<span class="p">-</span><span class="k">scg</span><span class="p">-</span>domain<span class="p">-</span>classifieri <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-gen-scg-domain-classifier"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>gen<span class="p">-</span>short<span class="p">-</span>readsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-gen-short-reads"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span><span class="nb">get</span><span class="p">-</span>collection<span class="p">-</span>infoi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-get-collection-info"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span><span class="nb">get</span><span class="p">-</span>coverage<span class="p">-</span>from<span class="p">-</span>bami <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-get-coverage-from-bam"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span><span class="nb">get</span><span class="p">-</span>hmm<span class="p">-</span>hits<span class="p">-</span>per<span class="p">-</span>gene<span class="p">-</span>calli <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-get-hmm-hits-per-gene-call"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span><span class="nb">get</span><span class="p">-</span>primer<span class="p">-</span>matchesi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-get-primer-matches"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>merge<span class="p">-</span>collectionsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-merge-collections"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>permute<span class="p">-</span>trnaseq<span class="p">-</span>seedsi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-permute-trnaseq-seeds"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>pfam<span class="p">-</span>accessions<span class="p">-</span><span class="k">to</span><span class="p">-</span>hmms<span class="p">-</span>directoryi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-pfam-accessions-to-hmms-directory"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>predict<span class="p">-</span>CPR<span class="p">-</span>genomesi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-predict-CPR-genomes"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>process<span class="p">-</span>genbanki <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-process-genbank"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>process<span class="p">-</span>genbank<span class="p">-</span>metadatai <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-process-genbank-metadata"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>reformat<span class="p">-</span>fastai <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-reformat-fasta"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>run<span class="p">-</span>eggnog<span class="p">-</span>mapperi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-run-eggnog-mapper"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>snvs<span class="p">-</span><span class="k">to</span><span class="p">-</span>interactivei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-snvs-to-interactive"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>tabulatei <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-tabulate"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>transpose<span class="p">-</span>matrixi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-transpose-matrix"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>variability<span class="p">-</span><span class="k">to</span><span class="p">-</span>vcfi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-variability-to-vcf"</span> %<span class="p">}</span>
inoreabbrev anvi<span class="p">-</span>script<span class="p">-</span>visualize<span class="p">-</span><span class="k">split</span><span class="p">-</span>coveragesi <span class="p">{</span>% <span class="nb">include</span> PROGRAM name<span class="p">=</span><span class="s2">"anvi-script-visualize-split-coverages"</span> %<span class="p">}</span>

</code></pre></div></div>

<p>Just for posterity, this is how I generated the data above if you ever wish to update it.</p>

<p>For artifacts, I run this (assuming that you have the anvi’o codebase at <code class="language-plaintext highlighter-rouge">~/github/anvio/</code>):</p>

<div class="language-bash highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
<span class="k">for </span>i <span class="k">in</span> <span class="sb">`</span><span class="nb">grep</span> <span class="s1">'{'</span> ~/github/anvio/anvio/docs/__init__.py | <span class="nb">grep</span> <span class="s1">'"'</span> | <span class="nb">awk</span> <span class="s1">'BEGIN{FS="\""}{print $2}'</span><span class="sb">`</span>
<span class="k">do
    </span><span class="nb">echo </span>inoreabbrev <span class="k">${</span><span class="nv">i</span><span class="k">}</span>i <span class="o">{</span>% include ARTIFACT <span class="nv">name</span><span class="o">=</span><span class="se">\"</span><span class="nv">$i</span><span class="se">\"</span> %<span class="o">}</span>
<span class="k">done</span>

</code></pre></div></div>

<p>For programs, I run these two:</p>

<div class="language-bash highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
<span class="k">for </span>i <span class="k">in</span> <span class="sb">`</span><span class="nb">ls</span> ~/github/anvio/bin/anvi-<span class="k">*</span> ~/github/anvio/sandbox/anvi-<span class="k">*</span> | <span class="nb">grep</span> <span class="nt">-v</span> <span class="s1">'_'</span> | <span class="nb">grep</span> <span class="nt">-v</span> <span class="s2">"augustus"</span><span class="sb">`</span>
<span class="k">do
    </span><span class="nb">echo </span>inoreabbrev <span class="si">$(</span><span class="nb">basename</span> <span class="nv">$i</span><span class="si">)</span>i <span class="o">{</span>% include PROGRAM <span class="nv">name</span><span class="o">=</span><span class="se">\"</span><span class="si">$(</span><span class="nb">basename</span> <span class="nv">$i</span><span class="si">)</span><span class="se">\"</span> %<span class="o">}</span>
<span class="k">done</span>

</code></pre></div></div>


    </div> <!-- article content -->

    
      <hr />
      
    

    </div><!-- /.article-wrap -->

    
  </article>
</div><!-- /#index -->

<div class="footer-wrap">
  <footer>
    

<div class="flex-container" style="width: 100%;">

  <div class="flex-child">
    <img style="width: 70%; border: 0px; border-radius: 0px; background-color:#4c4c4c; padding: 10px;" src="/images/logo-company.png">

      <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3984.1110027022614!2d101.59607168425829!3d3.064988688945323!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x31cc4eb079376a33%3A0xd9226112d1ae5b9d!2sMonash%20University%20Malaysia!5e0!3m2!1sen!2smy!4v1716624959097!5m2!1sen!2smy" width="90%" height="90%" style="border:0; margin-top: 20px; margin-bottom: 20px;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>



  </div>
  
<!-- <div class="vl" style="height: 270px; margin-left: -10px;"></div> -->

  <div class="flex-child" style="color: #ffffff; font-size: 20px;">
    Fast Genomics Solutions (M) Sdn Bhd<br>
    Phone: +60178505024 (Azam)<br>
    Phone: +601160905857 (Zarul)<br>

    Website: fastgenomicssolutions.com
  </div>
  
</div>

<div style="color: #ffffff; font-size: 20px;">
  <span >&copy; 2025 Fast Genomics Solutions. Powered by <a href="http://jekyllrb.com" rel="nofollow" style="color: #ffffff;">Jekyll</a> and a derivation of <a href="http://mademistakes.com/minimal-mistakes/" rel="nofollow" style="color: #ffffff;">MM</a>.</span>

</div>


  </footer>
</div><!-- /.footer-wrap -->

<script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
<script>window.jQuery || document.write('<script src="http://localhost:4000/assets/js/vendor/jquery-1.9.1.min.js"><\/script>')</script>
<script src="http://localhost:4000/assets/js/scripts.min.js"></script>


<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-420484-15', 'auto');
  ga('require', 'linkid', 'linkid.js');
  ga('send', 'pageview');
</script>



</body>
</html>
